src/routes/api/index.js

```diff
@@ -15,6 +16,7 @@ router.use('/trips', commentRoute);
 router.use('/users', roleRoute);
 router.use('/location', locationRoute);
 router.use('/users', userRoute);
+router.use('/accomodations', accomodationRouter);
```

Comment:
I think you could have used `accommodations` instead of accomodation with single `m`. This is because different routes will be pointing to accommodation with `double m`.